ons_alpha/jinja2/component_overrides/document-list/_macro.njk

```diff
@@ -4,19 +4,20 @@
         {% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{ ' ' }}{{ attribute }}="{{ value }}"{% endfor %}{% endif %}
     >
         {% for document in params.documents %}
-            {% set titleTag = params.titleTag | default("h2") %}
-            {% set openingTag = "<" + titleTag %}
-            {% set closingTag = "</" + titleTag + ">" %}
+            {% set titleTag = params.headingLevel | default(2) %}
```

Comment:
@zerolab / @ababic please note that the changes in this file are all copied from the design system (as we are temporarily overriding this component to make links optional) apart from where I have added a 'temporary change' comment.